### PR TITLE
Add more information about riskiness

### PR DESCRIPTION
**Pull request by beshaya**
Clarifies in the text that each Ibashan gets 3,000 microCovids to spend.
Adds some death rate equivalents to help people decide their budget.

### CODE REVIEW

**Comment by beshaya** [comment]
Related to #319

File: src/posts/paper/2-riskiness.ts
Comment (blanchardjeremy) [inline src/posts/paper/2-riskiness.ts:101]: Base jumping how often for 16 years?

Comment (beshaya) [reply src/posts/paper/2-riskiness.ts]: Updated to # of jumps.

**Review by blanchardjeremy — APPROVED** [review]
Thumbs up from me but I think someone with a better grasp on the science should review it before merging.

**Comment by catherio** [comment]
pardon the delay but yes these are small enough and simple enough changes that it was fully appropriate not to get 👍  :) :)

**Pull request merged**
